WordPress/src/main/java/org/wordpress/android/ui/media/services/MediaUploadService.java

```diff
@@ -50,6 +50,12 @@
 
     private static List<MediaModel> mPendingUploads = new ArrayList<>();
     private static List<MediaModel> mInProgressUploads = new ArrayList<>();
+    // we need this last map so to be able to update the PostModel once with all completed uploads,
+    // as opposed to editing the Post each time a single upload completes.
+    // Also, we need to process each Post's media in a batch, so that's why we are using a map
+    // instead of using a simple List (a map will have a List of MediaModel for their corresponding
+    // PostId where they're supposed to belong to as per the user intent)
+    private static HashMap<Integer, List<MediaModel>> mCompletedUploads = new HashMap<>();
```

Comment:
What do you think about calling this `mCompletedUploadsByPost` to make its meaning clearer?